snapshots/windows/windows.go

```diff
@@ -125,17 +125,21 @@ func (s *snapshotter) Usage(ctx context.Context, key string) (snapshots.Usage, e
 	if err != nil {
 		return snapshots.Usage{}, err
 	}
-	defer t.Rollback()
+	id, info, usage, err := storage.GetInfo(ctx, key)
+	t.Rollback() // transaction no longer needed at this point.
 
-	_, info, usage, err := storage.GetInfo(ctx, key)
 	if err != nil {
 		return snapshots.Usage{}, err
 	}
 
+	path := s.getSnapshotDir(id)
```

Comment:
nit: `path` is needed only for active snapshots, can be moved inside `if` block.

Reply:
Done.

Updated overlay.go as well, that's where I copy the code from. :)